a10_neutron_lbaas/v1/handler_pool.py

```diff
@@ -114,8 +67,8 @@ def stats(self, context, pool_id):
         pool = {'id': pool_id, 'tenant_id': tenant_id}
         with a10.A10Context(self, context, pool) as c:
             try:
```

Comment:
Considering an exception could be due to any number of reasons, do we want to log the exception?  Else we just reutrn the expected JSON structure with 0 values and the problem itself (which could be major or minor) goes complete unnoticed.  Unless that's the intended functionality.